spec/index.bs

```diff
@@ -6999,7 +6987,7 @@ All plain color formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/
 
 Only formats with {{GPUTextureComponentType}} {{GPUTextureComponentType/"float"}} can be blended.
 
-The {{GPUTextureUsage/STORAGE|GPUTextureUsage.STORAGE}} column specifies the support for {{GPUTextureUsage/STORAGE}} usage in the core API, including both {{GPUBindingType/"readonly-storage-texture"}} and {{GPUBindingType/"writeonly-storage-texture"}}. If there is a &star; mark, the atomic operations on the storage are also supported for this format.
+The {{GPUTextureUsage/STORAGE|GPUTextureUsage.STORAGE}} column specifies the support for {{GPUTextureUsage/STORAGE}} usage in the core API, including both {{GPUBindingType/"readonly-storage-texture"}} and {{GPUBindingType/"writeonly-storage-texture"}}.
```

Comment:
Out of curiosity, why was the notion of atomic texture ops removed?

Reply:
Because MSL doesn't have this even on the type level :(